tools/webpack-core/src/plugins/CopyExternalsPlugin.ts

```diff
@@ -12,6 +12,7 @@ import { getAppRelativePath, getSourcePosition, paths } from "../utils/paths";
 const { builtinModules } = require("module");
 const WebpackError = require("webpack/lib/WebpackError");
 const ModuleDependencyWarning = require("webpack/lib/ModuleDependencyWarning");
+const resolveRecursively = require("../utils/resolve-recurse/resolve.js");

```

Comment:
Can we just call this something like `getAllDependencies`? Just seems like the recursion there is an implementation detail and it's easy to confuse with `recurseDependencies` below.

Reply:
Changed